workflow/infrastructure-monitoring/container-monitoring.mdx

```diff
@@ -1,73 +1,131 @@
+        </Accordion>
+
+        <Accordion title="StatefulSet">
+            StatefulSets are used for managing stateful applications that are comprised of unique network identifiers, persistent storage, and specific ordering requirements for their deployment and scaling. 
+        </Accordion>
+
+        </AccordionGroup>
+
+</Accordion>
+
+# Usage Graph
+
+The Usage graph displays all of your team's Infrastructure usage data. Gain immediate insight into your stack's performance.
+
+### Change Usage Type
+
+Toggle between your Infrastructure CPU and Memory usage over time with the Usage Graph.
+
+<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExaDRkZHlldGN3bWJ4b281emh3Ym52ZHFpaGJ4anF2OWtzZTMyMW9vbiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/CMxUd2XqN2sQKFTXEW/giphy.gif" width="750" height="750"/>
+
+### Select Date Range
+
+Select your own date range from the calendar or choose from our default options.
+
+<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExMWpuamp5amFsM3dtazFobXVibnhpb2xmc3NqOWN6d3djczM4anQydCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/9ZT90BmYhPrqB3fiWt/giphy.gif" width="750" height="750"/>
+
+# Data Table
+
+The List Tab displays all of your specific Infrastructure data in a single table. You can manage your list view by adding and removing specific metrics.
+
+### Add & Remove Data
+Select specific Infrastructure data you would like to see in your data table by toggling on and off each metric.
+
+<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExNzhsdm1ycHB5b244a3B1ZTJnaDNjOGlmZzFteDFob2FrZjAzdG1nZiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/PtGflEPrP0lFAGdV4i/giphy.gif" width="750" height="750"/>
+
+### Filter & Search
+Access specific Hosts, Host IDs, Host Names, K8s Nodes, OS Types, and other key attributes with the Filters bar. Quickly index across all of your infrastructure data in one search. 
+
+<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExZXJmNDM5d2JmYzd4aTJ2OHg3ZHd3dms2bTJwZ3R1Z3dzcXl0anNkeiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/rEJAUGmdTiJZqvOEBT/giphy.gif" width="750" height="750"/>
+
+### Live & History View
+Toggle between a static data table with the History View or track your Infrastructure data in real-time with Live View.
+
+<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExOXR2ZmFlN244YndlcmYybTUyZXltZ3E5NXpxbXBrbHQ0MDU2cHE4MSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/Hm44urt27NsM3QDptx/giphy.gif" width="750" height="750"/>
+
+# Drilling Into Your Data
+
+The below section describes some of the many ways you can drill into your Infrastructure data. Learn how to pinpoint and address your major performance bottlenecks or issues, ensuring optimal system operation. Identify common usage patterns throughout your stack to ensure future system reliability and effective capacity planning.
```

Comment:
The second and third sentence in this feel like filler rather than explicit definition. Use the same framework of "what the section is. How it works. what you can do with it." mentioned above. For example,

"(your first sentence). By default Middleware connects data across your stack using span and thread ids to ensure a drillable experience across all components. This enables users to have an easy path to root cause regardless of where they start their exploration. In the examples below you'll see how to drill into the Host List and Kubernetes Data, however the drill is available from any data point across the Data table."